files/en-us/mozilla/add-ons/webextensions/add_a_button_to_the_toolbar/index.html

```diff
@@ -204,7 +204,7 @@ <h2 id="Learn_more">Learn more</h2>
   <ul>
    <li><a href="https://github.com/mdn/webextensions-examples/tree/master/beastify">beastify</a></li>
    <li><a class="external external-icon" href="https://github.com/mdn/webextensions-examples/tree/master/bookmark-it">Bookmark it!</a></li>
-   <li><a class="external external-icon" href="https://github.com/mdn/webextensions-examples/tree/master/favourite-colour">favourite-colour</a></li>
+   <li><a class="external external-icon" href="https://github.com/mdn/webextensions-examples/tree/master/favorite-colour">favorite-colour</a></li>
```

Comment:
This is the working link
```suggestion
   <li><a class="external external-icon" href="https://github.com/mdn/webextensions-examples/tree/master/favourite-colour">favorite-colour</a></li>
```